test/e2e/fixture/applicationsets/utils/fixture.go

```diff
@@ -114,7 +113,7 @@ func EnsureCleanState(t *testing.T) {
 			// Fail
 			msg := fmt.Sprintf("Waiting for list of ApplicationSets to be size zero: %d", len(list.Items))
 			// Intentionally not making this an Errorf, so it can be printf-ed for debugging purposes.
-			return errors.New(msg)
+			return fmt.Errorf(msg)
```

Comment:
Should we clean this (and other ones in this file) up to combine them into a single fmt.Errorf() call?  If we debug with printf, we would've to edit it anyway.

Reply:
Agreed, will update!